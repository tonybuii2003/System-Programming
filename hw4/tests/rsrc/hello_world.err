[00000.000000] STARTUP
[00000.000027] PROMPT
[00000.100148] INPUT run echo hello world
[00000.100238] CHANGE 363922: none -> running
[00000.100280] PROMPT
[00000.100482] SIGNAL 17
[00000.100501] CHANGE 363922: running -> stopped
[00000.200488] INPUT cont 0
[00000.200540] CHANGE 363922: stopped -> running
[00000.200556] PROMPT
hello world
[00000.201531] SIGNAL 17
[00000.201561] CHANGE 363922: running -> dead
[00000.300786] INPUT quit
[00000.300839] SHUTDOWN
