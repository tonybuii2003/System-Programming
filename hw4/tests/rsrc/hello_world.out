0	363922	T	running		echo hello world
0	363922	T	stopped		echo hello world
0	363922	T	running		echo hello world
0	363922	T	dead	0x0	echo hello world
